### PR TITLE
Sprint4 24 create a publisher factoryseeder

### LINKED ISSUE #24: Create a publisher factory/seeder
Blocked by #22 

We need a factory and database seeder for the publisher table so we have data for populating the dropdown when adding a new game.

## Acceptance criteria:
- A PublisherFactory class (don't really need to do anything with this yet)
- A PublisherSeeder class that inserts some publisher records (name and code):
  - Fantasy Flight Games (FFG)
  - Z-Man Games (ZMG)
  - KOSMOS (KOS)
  - Days of Wonder (DOW)
  - Cool Mini Or Not (CMN)
  - Ravensburger (RVN)
  - Rio Grande Games (RGG)

### PR DESCRIPTION
**Pull request by lrapp1**
resolves #24 

What's done:
- Empty PublisherFactory class created
- PublisherSeeder used to insert specified data from issue into "publishers" table
- PublisherSeeder class called in DatabaseSeeder
- PublisherSeeder seems to insert data

### CODE REVIEW

**Review by 12bitsample — APPROVED** [review]
This all looks good to me, I wasn't able to find any major problems anyway! I would say so long as it is running properly in docker, it should be good to go, great job!

**Comment by mdigrandi** [comment]
Everything looks good to me! Good job!

File: app/Models/Publisher.php [hunk 1]
```diff
@@ -11,7 +12,13 @@ class Publisher extends Model
      */
     protected $fillable = ['id', 'publisher', ];
 
+public function publisher() {
```
Comment (oldskoolfan) [inline app/Models/Publisher.php:15]: this is not needed and can be removed

File: app/Models/Publisher.php [hunk 2]
```diff
@@ -11,7 +12,13 @@ class Publisher extends Model
...
+    return $this->has(Publisher::class);
+}
+
     public function games() {
         return $this->hasMany(Game::class);
     }
+
+    use HasFactory;
```
Comment (oldskoolfan) [inline app/Models/Publisher.php:23]: traits typically go at the top of the class

File: app/Models/Publisher.php (hunk 1 above)
Comment (lrapp1) [reply app/Models/Publisher.php]: This is done

File: app/Models/Publisher.php (hunk 2 above)
Comment (lrapp1) [reply app/Models/Publisher.php]: *clown

**Comment by lrapp1** [comment]
Suggestions implemented.

**Review by JessEpley-dev — APPROVED** [review]
Looks good, Laura!

**Review by oldskoolfan — APPROVED** [review]

**Pull request merged**